app/src/main/java/org/oppia/app/activity/ActivityComponent.kt

```diff
@@ -49,5 +51,7 @@ interface ActivityComponent {
   fun inject(stateFragmentTestActivity: StateFragmentTestActivity)
   fun inject(storyActivity: StoryActivity)
   fun inject(topicActivity: TopicActivity)
+  fun inject(topicTestActivity: TopicTestActivity)
+  fun inject(topicTestActivity: TopicTestActivityForStory)
```

Comment:
Please rename it to `topicTestActvityForStory` both are having same names.

Reply:
Done